src/pages/index.js

```diff
@@ -46,15 +113,15 @@ const enableValidation = (selectors) => {
   });
 };;
 
-cardSection.renderItems(initialCards);
+
 enableValidation(selectors);
 profileEditForm.setEventListeners();
 addCard.setEventListeners();
 previewModal.setEventListeners();
 
-function updateUserInfo({ name, description }) {
-  currentUserInfo.setUserInfo({ name, description });
-}
+//function updateUserInfo({ name, description }) {
+//  currentUserInfo.setUserInfo({ name, description });
+//}
 
 function setFormInfo(nameSelector, detailsSelector) {
   const formName = document.querySelector(nameSelector);
```

Comment:
**Could be improved**

It’s better to find **constants** only 1 time at the top of the file so as not to waste resources on searching them again and again when you call a method (function) because searching is a very **hard** operation for the browser engine